src/components/templates/AddStory.js

```diff
@@ -106,6 +134,26 @@ const AddStory = ({ toggleModal }) => {
         Characters Left:
         {240 - newStory.content.length}
       </span>
+      <label htmlFor="story-add-content">
+        <div>
+          Tags
+        </div>
+        <input
+          type="text"
+          id="story-add-tags"
+          name="tags"
+          value={newStory.tags}
+          onChange={handleStoryContentChange}
+          onKeyUp={onKeyUp}
+        />
+      </label>
+      {tags ? (tags.map((tag) => (`#${tag} `))) : <></>}
```

Comment:
We can just use an AND operator for this

```suggestion
      { Boolean(tags) && tags.map((tag) => `#${tag} `) }
```